cincinnati/src/lib.rs

```diff
@@ -818,12 +818,36 @@ pub mod testing {
     use super::*;
     use crate::plugins::internal::versioned_graph::VersionedGraph;
 
-    pub fn generate_graph(include_conditional_edge: bool) -> Graph {
+    pub fn generate_graph(include_conditional_edge: bool, include_always_condition: bool) -> Graph {
         let mut graph = Graph::default();
 
+        if include_conditional_edge && include_always_condition {
+            graph.conditional_edges = Some(vec![ConditionalEdge {
+                edge_regex: ConditionalUpdateEdge {
+                    from: "1.0.0".to_string(),
+                    to: "2\\.0\\.*".to_string(),
```

Comment:
This expression should probably be `2[.]0[.].*`.  `[.]` is easier for me to think about than stacking escapes with `\\.`.  And you need an un-escaped `.` for the `.*` portion.

Reply:
err, also, when we declare conditional risks in graph-data, we have a literal `to` and a regular expression `from`.  In this code at the moment, you have a literal `from` and a regular expression `to`, which seems backwards.

Reply:
fixed